.github/workflows/tag-release.yml

```diff
@@ -0,0 +1,40 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+permissions:
+  contents: write
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.21.1'
```

Comment:
The version that is used in the projet is 1.20, if that will have some side-effects(?)

Reply:
I thought go had a full backwards compatible compiler, I can change this back though.

Reply:
Nah, ur probably right on this one! :)